comp/dogstatsd/server/parse.go

```diff
@@ -21,6 +22,10 @@ const (
 	metricSampleType messageType = iota
 	serviceCheckType
 	eventType
+	// readerCacheExpiration determines the duration for which the cgroups data is cached in the cgroups reader.
+	// This value needs to be large enough to reduce latency and I/O load.
+	// It also needs to be small enough to catch the first traces of new containers.
```

Comment:
I think this comment should be adapted to dogstatsd.

Reply:
Got it, done 👍